heartbeat/docs/getting-started.asciidoc

```diff
@@ -0,0 +1,261 @@
+If you use Apt or Yum, you can {libbeat}/setup-repositories.html[install Heartbeat from our repositories] to update to the newest version more easily.
+
+See our https://www.elastic.co/downloads/beats/heartbeat[download page] for other installation options, such as 32-bit images.
+
+==================================================
+
+[[deb]]
+*deb:*
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+curl -L -O {downloads}/heartbeat/heartbeat-{version}-amd64.deb
+sudo dpkg -i heartbeat-{version}-amd64.deb
+----------------------------------------------------------------------
+
+[[rpm]]
+*rpm:*
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+curl -L -O {downloads}/heartbeat/heartbeat-{version}-x86_64.rpm
+sudo rpm -vi heartbeat-{version}-x86_64.rpm
+----------------------------------------------------------------------
+
+[[mac]]
+*mac:*
+
+["source","sh",subs="attributes"]
+------------------------------------------------
+curl -L -O {downloads}/heartbeat/heartbeat-{version}-darwin-x86_64.tar.gz
+tar xzvf heartbeat-{version}-darwin-x86_64.tar.gz
+------------------------------------------------
+
+[[win]]
+*win:*
+
+. Download the Heartbeat Windows zip file from the
+https://www.elastic.co/downloads/beats/heartbeat[downloads page].
+
+. Extract the contents of the zip file into `C:\Program Files`.
+
+. Rename the +heartbeat-<version>-windows+ directory to +Heartbeat+.
+
+. Open a PowerShell prompt as an Administrator (right-click the PowerShell icon
+and select *Run As Administrator*). If you are running Windows XP, you may need
+to download and install PowerShell.
+
+. Run the following commands to install Heartbeat as a Windows service:
++
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+PS > cd 'C:\Program Files\Heartbeat'
+PS C:\Program Files\Heartbeat> .\install-service-heartbeat.ps1
+----------------------------------------------------------------------
+
+NOTE: If script execution is disabled on your system, you need to set the
+execution policy for the current session to allow the script to run. For
+example:
++PowerShell.exe -ExecutionPolicy UnRestricted -File .\install-service-heartbeat.ps1+.
+
+Before starting Heartbeat, you should look at the configuration options in
+the configuration file, for example +C:\Program Files\Heartbeat\heartbeat.yml+
+or +/etc/heartbeat/heartbeat.yml+. For more information about these
+options, see <<heartbeat-configuration-details>>.
+
+[[heartbeat-configuration]]
+=== Step 2: Configuring Heartbeat
+
+To configure Heartbeat, you edit the configuration file. For rpm and deb,
+you'll find the configuration file at +/etc/heartbeat/heartbeat.yml+.
+For mac and win, look in the archive that you just extracted.
+
+Heartbeat uses monitors to configure ping behavior. You configure each
```

Comment:
@urso Will we also have "passive monitors" at some point (or maybe there's a different word)? At this point, I'm wondering if it makes sense to mention that we only support active monitors. WDYT?